promptsource/templates/khalidalt/HuffPost/templates.yaml

```diff
@@ -0,0 +1,96 @@
+dataset: khalidalt/HuffPost
+templates:
+  19d2449b-1a52-4079-aaec-301517ee6ee1: !Template
+    answer_choices: POLITICS|||  WELLNESS|||  ENTERTAINMENT|||  TRAVEL|||  STYLE &
+      BEAUTY|||  PARENTING|||  HEALTHY LIVING|||  QUEER VOICES|||  FOOD & DRINK|||  BUSINESS|||  COMEDY|||  SPORTS|||  BLACK
+      VOICES|||  HOME & LIVING|||  PARENTS|||  THE WORLDPOST|||  WEDDINGS|||  WOMEN|||  IMPACT|||  DIVORCE|||  CRIME|||  MEDIA|||  WEIRD
+      NEWS|||  GREEN|||  WORLDPOST|||  RELIGION|||  STYLE|||  SCIENCE|||  WORLD NEWS|||  TASTE|||  TECH|||  MONEY|||  ARTS|||  FIFTY|||  GOOD
+      NEWS|||  ARTS & CULTURE|||  ENVIRONMENT|||  COLLEGE|||  LATINO VOICES|||  CULTURE
+      & ARTS|||  EDUCATION
+    id: 19d2449b-1a52-4079-aaec-301517ee6ee1
+    jinja: 'What can you infer from the headline and short description given about
+      the news category? Headline: {{headline}}, Short description: {{short_description}}  |||
+      {{answer_choices[label]}}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: true
+      metrics:
+      - Accuracy
+      original_task: true
+    name: infer_from
+    reference: ''
+  21973a70-f643-4172-a5f3-624aaf0d339a: !Template
+    answer_choices: POLITICS|||  WELLNESS|||  ENTERTAINMENT|||  TRAVEL|||  STYLE &
+      BEAUTY|||  PARENTING|||  HEALTHY LIVING|||  QUEER VOICES|||  FOOD & DRINK|||  BUSINESS|||  COMEDY|||  SPORTS|||  BLACK
+      VOICES|||  HOME & LIVING|||  PARENTS|||  THE WORLDPOST|||  WEDDINGS|||  WOMEN|||  IMPACT|||  DIVORCE|||  CRIME|||  MEDIA|||  WEIRD
+      NEWS|||  GREEN|||  WORLDPOST|||  RELIGION|||  STYLE|||  SCIENCE|||  WORLD NEWS|||  TASTE|||  TECH|||  MONEY|||  ARTS|||  FIFTY|||  GOOD
+      NEWS|||  ARTS & CULTURE|||  ENVIRONMENT|||  COLLEGE|||  LATINO VOICES|||  CULTURE
+      & ARTS|||  EDUCATION
+    id: 21973a70-f643-4172-a5f3-624aaf0d339a
+    jinja: 'Given the following news headline: {{headline}}. What is a correct category
```

Comment:
Why doesn't this prompt also use the short description? Is the original task intended to just use the headline or both?